src/features/network/store.ts

```diff
@@ -5,12 +5,16 @@ import { replacer, reviver } from "shared/helpers"
 import { NetworkId, NetworkParams, NetworksState } from "./types"
 
 const initialState = {
-  activeId: 0,
-  nextId: 1,
-  byId: new Map([[0, { name: "Manifest", url: "/api" }]]),
+  nextId: 2,
+  byId: new Map([
+    [0, { name: "Manifest Ledger", url: "/api", filter: "alberto" }],
+    [1, { name: "GBOP Ledger", url: "/api-end", filter: "end-labs" }],
```

Comment:
@jgryffindor Please double-check the `url` here. I'm pretty sure it's what I heard you say. Easy to change.

Reply:
Looks good!